### PR TITLE
💄 #176 create the consultancy hero section

### LINKED ISSUE #176: 🎨 Create the consultancy hero section
#  Create the consultancy hero section 
---

Quick links: [Summary](#summary) | [Description](#description) | [Technical-Analysis](#Technical-Analysis) | [Acceptance-Criteria](#Acceptance-Criteria)

---

### Summary

As a user, I want a nice hero section on the consultancy page that captures my attention and introduces the page content. Reuse the text-content component created in #20, and the reusable slider button in  #24. I want a good experience on the web and phone.

Design:

consultancy hero section

Link to full design:

```
https://xd.adobe.com/view/b98caf27-6549-49e5-8ed8-c5234788df61-ecdf/screen/cc35e19a-24f2-42ce-8803-ee2942839528/
```
 
### Technical Analysis

This feature can be found in libs/features/pages/consultancy-page inside a components folder. The component's name is consultancy-hero-section.

### Acceptance Criteria

1. When rendered the component is pixel perfect compared to the design above.
2. The component is clean and does not have more than 30 lines
3. The component SCSS uses CSS variables for colour, font, ... Connected to the global style.scss
5. The design is responsive for screen, tablet and phone

### PR DESCRIPTION
**Pull request by jbdura**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Reference the issue related to this PR as shown below. Every issue has it's own number you can find the issue numbers [here](https://github.com/italanta/italanta-apps/issues).

Fixes #176 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Screenshot (optional)
small
Screenshot from 2023-09-06 15-37-01
medium
Screenshot from 2023-09-06 15-36-30
large
Screenshot from 2023-09-06 15-36-15
## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/features/pages/consultancy-page/src/lib/components/consultancy-hero-section/consultancy-hero-section.component.scss [hunk 1]
```diff
@@ -0,0 +1,93 @@
+.container {
+  top: 0px;
+  left: 0px;
+  width: 100vw;
+  height: 100vh;
+  background: #DBC2F5 0% 0% no-repeat padding-box;
```
Comment (AnnaL001) [inline libs/features/pages/consultancy-page/src/lib/components/consultancy-hero-section/consultancy-hero-section.component.scss:6]: For reuse of colors, use CSS variables specified in elements/theming/_variables.scss for specifying colors/background colors.

File: libs/features/pages/consultancy-page/src/lib/components/consultancy-hero-section/consultancy-hero-section.component.scss [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        /* For mobile, further reduce the size of the circles */
+        height: 200px;
+        width: 200px;
+      }
+    }
+
+    .circ-1 {
+      top: 229px;
+      left: 219px;
+      background: transparent linear-gradient(90deg, #DBC2F500 0%, #DBC2F5E6 100%) 0% 0% no-repeat padding-box;
```
Comment (AnnaL001) [inline libs/features/pages/consultancy-page/src/lib/components/consultancy-hero-section/consultancy-hero-section.component.scss:73]: Same here, for linear gradient

File: libs/features/pages/consultancy-page/src/lib/components/consultancy-hero-section/consultancy-hero-section.component.scss [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    }
+
+    .circ-2 {
+      top: 229px;
+      left: 846px;
+      background: transparent linear-gradient(90deg, #DBC2F500 0%, #DBC2F5E6 100%) 0% 0% no-repeat padding-box;
```
Comment (AnnaL001) [inline libs/features/pages/consultancy-page/src/lib/components/consultancy-hero-section/consultancy-hero-section.component.scss:79]: Same here

File: libs/features/pages/consultancy-page/src/lib/components/consultancy-hero-section/consultancy-hero-section.component.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    `
+  }
+
+  buttonData = {
+    text: 'Unlock your potential ', // Fill the require
+    bgColor: 'black',
+    color: 'white',
+    hoverBgColor: 'transparent',
+  }
+
```
Comment (AnnaL001) [inline libs/features/pages/consultancy-page/src/lib/components/consultancy-hero-section/consultancy-hero-section.component.ts:25]: To keep the component lean and ensure readability and maintainability, the 'herotexts' and 'buttonData' variable content can be moved to a Typescript file in the data module

**Review by AnnaL001 — CHANGES_REQUESTED** [review]
Nicely done, just a couple of changes

**Comment by jbdura** [comment]
All recommended fixes completed

**Comment by LemmyMwaura** [comment]
Closing this as it's already been implemented

**Pull request closed without merging**